Client/Frontend/Settings/SettingsViewController.swift

```diff
@@ -225,25 +225,29 @@ class SettingsViewController: TableViewController {
             BoolRow(title: Strings.Block_all_cookies, option: Preferences.Privacy.blockAllCookies, onValueChange: { [unowned self] in
                 func toggleCookieSetting(with status: Bool) {
                     //Lock/Unlock Cookie Folder
-                    let success = FileManager.default.setFolderAccess([
-                        (.cookie, status),
-                        (.webSiteData, status)
-                        ])
-                    if success {
-                        Preferences.Privacy.blockAllCookies.value = status
-                    } else {
-                        //Revert the changes. Not handling success here to avoid a loop.
-                        _ = FileManager.default.setFolderAccess([
-                            (.cookie, false),
-                            (.webSiteData, false)
+                    let completionBlock: (Bool) -> Void = {(_) in
```

Comment:
```suggestion
                    let completionBlock: Bool -> Void = { _ in
```
(remove two extraneous sets of `()`s